automata/regex/parser.py

```diff
@@ -243,6 +276,17 @@ def op(self, left, right):
         return left
 
 
+class ShuffleToken(InfixOperator):
+    """Subclass of infix operator defining the shuffle operator."""
+
+    def get_precedence(self):
```

Comment:
@eliotwrobson Not necessary for this PR, but what do you think about making `precedence` a static property on the class, rather than a method? When I search through the codebase, most of these `get_precedence()` functions return integers.

Something like:

```python
class ShuffleToken(InfixOperator):
    """Subclass of infix operator defining the shuffle operator."""

    precedence = 1
```

But yeah, what do you think?

Reply:
@eliotwrobson I see. Sounds good to me.